internal/gui/gui.go

```diff
@@ -64,13 +64,13 @@ var (
 )
 
 // GetOnGUIReady creates func to run on GUI startup.
-func GetOnGUIReady(icon []byte, conf *visorconfig.V1) (ret func()) {
+func GetOnGUIReady(icon []byte, conf *visorconfig.V1) func() {
 	doneCh := make(chan bool, 1)
 	logger := logging.NewMasterLogger()
 	logger.SetLevel(logrus.InfoLevel)
 
 	httpC := getHTTPClient(conf, context.Background(), logger)
-
+	var ret func()
```

Comment:
No need to declare the variable here. We can just use `return func(){...}` directly inside the function, right?

Reply:
rectified in [99e3dcc](https://github.com/skycoin/skywire/pull/1155/commits/99e3dcc4982b68229c90378ff5e55c10f8793828)